ktor-client/ktor-client-core/posix/src/io/ktor/client/utils/CoroutineUtilsPosix.kt

```diff
@@ -15,3 +16,16 @@ public actual fun Dispatchers.clientDispatcher(
     threadCount: Int,
     dispatcherName: String
 ): CoroutineDispatcher = Unconfined
+
+internal actual fun checkCoroutinesVersion() {
+    try {
+        val parent = Job()
+        parent.freeze()
+        Job(parent)
+    } catch (cause: Throwable) {
+        val message = "Invalid coroutines version. " +
```

Comment:
The word "invalid" is not perfect here. I would say something like "Native ktor HttpClient requires  kotlinx.coroutines version with ....." or "kotlinx.coroutines version with .... is required for ktor native"

Reply:
+1. I think the more help we provide folks, the better

Reply:
Thanks, will fix